### DISCUSSION TITLE
CarPlay support

### DISCUSSION THREAD

**Discussion by Luis-Lourenco** [advplyr/audiobookshelf#2625] [Ideas]
Good evening,

Do you think it's possible to add support for CarPlay? I try to add my Podcasts directory to plex, and it's horrible job. 
I can set play on my screen, but it was ver nice to have this support. 

 I don't know what requisites need to support. 

Thank you.

**Comment by nichwall** [comment]
Duplicate of https://github.com/advplyr/audiobookshelf-app/discussions/81

Actual issue tracking it (lots of duplicates): https://github.com/advplyr/audiobookshelf-app/issues/475

The best way to show support for a feature is to leave a "thumbs up" reaction to the top comment, not a comment that just says "+1" or "I want this".

**Comment by jeeftor** [comment]
Plappa has this -> its what I use + tailscale
